test/public/runs/detail.test.js

```diff
@@ -143,9 +139,8 @@ module.exports = () => {
     it('successfully update detectors qualities in EDIT mode', async () => {
         await reloadPage(page);
         await pressElement(page, '#edit-run');
-        await waitForTimeout(100);
         await pressElement(page, '#Run-detectors .dropdown-trigger');
-        await waitForTimeout(100);
+
```

Comment:
You need to wait for something here

Reply:
Done

Reply:
I think that dropdown is too generic, you might catch another dropdown. You need to check that the dropdown that correspond to the trigger you clicked is visible (`getPopoverSelector`)

Reply:
Done